ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -60,6 +76,32 @@ void drawTextInput(SkCanvas *canvas,
   // paintParagraph
   yOffset = (layout.getContentFrame().size.height - paragraph->getHeight()) / 2;
   paragraph->paint(canvas, frame.origin.x + layout.contentInsets.left, frame.origin.y + layout.contentInsets.top + yOffset);
+
+  // draw Cursor
+  position = cursor_.end - cursor_.locationFromEnd;
+  if (cursor_.locationFromEnd == cursor_.end) {
+    rects = paragraph->getRectsForRange(0, position+1, RectHeightStyle::kTight, RectWidthStyle::kTight);
+    cursorRect.fLeft =  frame.origin.x + rects[0].rect.left() + layout.contentInsets.left;
+    cursorRect.fRight = frame.origin.x + rects[0].rect.left() + layout.contentInsets.left;
+  }
+  else {
+    rects = paragraph->getRectsForRange(0, position, RectHeightStyle::kTight, RectWidthStyle::kTight);
+    cursorRect.fLeft =  frame.origin.x + rects[0].rect.right() + layout.contentInsets.left;
+    cursorRect.fRight = frame.origin.x + rects[0].rect.right() + layout.contentInsets.left;
+  }
+  cursorRect.fTop = frame.origin.y + yOffset + layout.contentInsets.top;
+  cursorRect.fBottom = frame.origin.y + yOffset + layout.contentInsets.top + paragraph->getHeight();
+  
+  if (props.selectionColor) {
+    cursorPaint.setColor(RSkColorFromSharedColor(props.selectionColor, SK_ColorTRANSPARENT));
```

Comment:
instead of TRANSPARENT , can give directly blue color. So we will not require the if check here